### DISCUSSION TITLE
Setting up a site . . . icons? Also, console.log() seems to break when recording newly inserted info

### DISCUSSION THREAD

**Discussion by JadedChara** [socketio/socket.io#4264] [Q&A]
I've fixed a few issues with one of my projects, but I am wondering how to implement, if at all possible, a website icon. Additionally, when data gets pushed via client side into the `players` variable, I have a setInterval that clears and logs the data. It worked previously, but now when I run it, it returns the first socket's ID repeatedly, the data, then clears everything after that initial ID, rather than the whole group. I'm not sure if it's just a variable definition error, or if it's some other issue. The thing effectively looks like this :

```js
{
"id": {
//loop breaks...
{
"id": {
{
"id": {
//...And will continue repeating these two lines, before just logging the info itself
  "some random data"},
"Other id":{
  "Some more random data""}
}

```

I am genuinely stumped as to why it's doing this. For clarification, I'm compiling everything via Repl.it, if that's any use.

**Comment by darrachequesne** [comment]
Hi! Could you please provide a simple example reproducing the issue? You can use [Replit](https://replit.com/@socketio/socketio-minimal-example), [StackBlitz](https://stackblitz.com/edit/socketio-base?file=index.js) or [CodeSandbox](https://codesandbox.io/s/socket-io-minimal-example-k3h2l), or fork the following repository: https://github.com/socketio/socket.io-minimal-example

Thanks in advance!

**Comment by JadedChara** [comment]
Actually, I've got a project that uses this as a dependency! Here it is: [W.I.P. Terminus Game API](https://replit.com/@Nightstrike/Terminus-game-dev-1). I've been basically giving myself a crash course in Node and managing a Github page, so not everything is exactly . . . best practice. It's mainly intended just as a fun project, though.
